docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
+- An account on the [Viam app](https://app.viam.com)
+
+First, follow the [instructions to create a new machine](/cloud/machines/#add-a-new-machine) representing your computer.
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+
+You need to configure the following {{< glossary_tooltip term_id="resource" text="resources" >}} to detect people with a webcam:
+
+1. A [`webcam` camera](/components/camera/webcam/)
+2. An [`tflite_cpu` ML model service](/services/ml/deploy/tflite_cpu/)
+3. An [`mlmodel` computer vision service](/services/vision/mlmodel/)
+
+Follow these instructions to configure your machine and test detections:
+
+{{%expand "Step 1: Configure your webcam" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `camera` type, then select the `webcam` model.
+Enter a name or use the suggested name for your camera and click **Create**.
+
+Click the **Save** button in the top right corner of the page to save your config.
+Go to the **CONTROL** tab and expand the camera's remote control card to test the camera stream.
+If you don't see an image stream, you need to [configure the `video_path` attribute](/components/camera/webcam/#using-video_path).
+
+For more detailed information, including optional attribute configuration, see the [`webcam` docs](/components/camera/webcam/).
```

Comment:
If you like, the lines preceeding this line are all stepwise instructions: they might look good as bullet points within this expander. Maybe something like this:

```
- Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com/).
- Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
- Select the `camera` type, then select the `webcam` model.
- Enter a name or use the suggested name for your camera and click **Create**.
- Click the **Save** button in the top right corner of the page to save your config.
- Go to the **CONTROL** tab and expand the camera's remote control card to test the camera stream.
   If you don't see an image stream, you need to [configure the `video_path` attribute](/components/camera/webcam/#using-video_path).

For more detailed information, including optional attribute configuration, see the [`webcam` docs](/components/camera/webcam/).
```

If you like this approach and use it for this expander, suggest expanding this practices to the other ... expanders. If you don't like this approach, and want to keep it conversational, consider using segue words between some sentence so they are not all imperative directives: "Next ... " "First", etc. Minor!